build.gradle

```diff
@@ -104,6 +111,15 @@ allprojects {
       ) {
         entry 'camunda-engine'
       }
+      // Solves CVE-2022-31159
```

Comment:
We seem to never have had that in the suppressions. Add a bit to this comment to state that this is due to the discrepancy of CVE sources. Something like:

`// Solves CVE-2022-31159, happens only on local builds when running dependencyCheck gradle task`